data/mods/ssb/abilities.ts

```diff
@@ -104,6 +104,21 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 		},
 	},
 
+	// Cake
+	notenoughremoval: {
+		shortDesc: "Sets hazards at the end of every turn.",
+		name: "Not Enough Removal",
+		onResidualOrder: 28,
+		onResidualSubOrder: 2,
+		onResidual(pokemon) {
+			const hazard = this.sample(['stealthrock', 'spikes', 'toxicspikes', 'stickyweb', 'gmaxsteelsurge']);
+			this.add('-activate', pokemon, 'ability: Not Enough Removal');
+			for (const side of this.sides) {
+				side.addSideCondition(hazard);
+			}
```

Comment:
should there be better handling of trying to add hazards that are already there? right now it will show the ability message and then just do nothing, which is probably fine just confusing

Reply:
I can handle it just like intim yeah